ci/env/install-dependencies.sh

```diff
@@ -517,13 +517,28 @@ install_thirdparty_packages() {
   CC=gcc python -m pip install psutil==5.9.6 setproctitle==1.2.2 colorama==0.4.6 --target="${RAY_THIRDPARTY_FILES}"
 }
 
+install_zstd() {
```

Comment:
Thanks for the PR; is this now a hard requirement for running ray? If so, perhaps it should be included within the ray wheel itself instead of a requirement on the environment?

Reply:
Thanks for the suggestion! Yes it is. To build raylet we would need the library to be installed first tho.

Could you please point me to where I should put the installation cmds? or if there is any similar change in the past I could take a look?

Reply:
i would assume it's somewhere in BUILD.bazel, but @fishbone  would know much better here

Reply:
also unsure about including a non-standard cpp libraries as part of ray vs. having it as a plugin/runtime requirement, but also leave it for Yi who is on the core team

Reply:
i updated to use bazel to build zstd instead, which seems cleaner than previous approach.